horreum-backend/src/main/java/io/hyperfoil/tools/horreum/changedetection/HunterEDivisiveModel.java

```diff
@@ -0,0 +1,275 @@
+
+        if (results.size() > 3) {
+            Iterator<String> resultIter = results.iterator();
+            while (resultIter.hasNext()) {
+                String line = resultIter.next();
+                if (line.contains("··")) { //this is dynamic depending on the number of significant figures from the metric, we print it to 2sf, so 2x '.' should suffice
+
+                    String change = resultIter.next().trim();
+                    resultIter.next(); // skip line containing '··'
+                    String nextLine = resultIter.next().replaceAll("( )+", " ");
+                    String[] changeDetails = nextLine.split(" ");
+                    String timestamp = changeDetails[0].concat(" ").concat(changeDetails[1]).concat(" ").concat(changeDetails[2]);
+                    Integer datapointID = changeDetails[3] == "" ? 0 :  Integer.parseInt(changeDetails[3]);
+
+                    log.debugf("Found changepoint `%s` at `%s` for dataset: %d", change, timestamp, datapointID);
+
+                    Optional<DataPointDAO> foundDataPoint = changePointSupplier.apply(datapointID);
+
+                    if (foundDataPoint.isPresent()) {
+                        ChangeDAO changePoint = ChangeDAO.fromDatapoint(foundDataPoint.get());
+
+                        //TODO: this should be templated
+                        changePoint.description = String.format("eDivisive change `%s` at `%s` for dataset: %d", change, timestamp, datapointID);
+
+                        log.trace(changePoint.description);
+                        changeConsumer.accept(changePoint);
+                    } else {
+                        log.errorf("Could not find datapoint in set!");
+                    }
+                }
+            }
+
+        } else {
+            log.debugf("No change points were detected in : %s", tmpFiles.tmpdir.getAbsolutePath());
+        }
+    }
+
+    protected boolean validateInputCsv(TmpFiles tmpFiles) {
+        executeProcess(tmpFiles, true, "bash", "-l", "-c", "hunter validate");
+
+        try(FileReader fileReader = new FileReader(tmpFiles.logFile);
+            BufferedReader reader = new BufferedReader(fileReader);){
+
+            Optional<String> optLine = reader.lines().filter(line -> line.contains("Validation finished")).findFirst();
+            if(optLine.isEmpty()) {
+                log.errorf("Could not validate: %s", tmpFiles.tmpdir.getAbsolutePath());
+                return false;
+            }
+            if( optLine.get().contains("INVALID") ) {
+                log.errorf("Invalid format for: %s; see log for details: %s", tmpFiles.tmpdir.getAbsolutePath(), tmpFiles.logFile.getAbsolutePath());
+                return false;
+            }
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException(e);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
If we simply forward the exception as `RuntimeException` we don't need `FileNotFound` catch clause.
```suggestion
        } catch (IOException e) {
            throw new RuntimeException(e);
        }
```